css/src/components/hero.scss

```diff
@@ -34,6 +34,52 @@ $hero-image-width-widescreen: calc((100% - 2 * $layout-widescreen-gap) * 0.55) !
 		}
 	}
 
+	.hero-details {
+		display: flex;
+		flex-grow: 1;
+		padding-block: $hero-padding-xs;
+
+		@include tablet {
+			justify-content: flex-end;
+			padding-block: $hero-padding-sm;
+			padding-inline-start: $hero-padding-xs;
+		}
+
+		@include desktop {
+			width: $hero-content-fluid-width;
+			min-height: $hero-min-height-default;
+			padding-block: $hero-padding-default;
+			padding-inline-start: $hero-padding-sm;
+			padding-inline-end: $hero-padding-sm;
+		}
+
+		.hero-details-card {
+			align-self: flex-start;
+			backdrop-filter: blur(20px);
+
+			.theme-high-contrast & {
+				background: unset;
+				background-color: var(--theme-body-background);
+				opacity: 1;
+			}
+
+			@include desktop {
+				background: linear-gradient(
+					116deg,
+					var(--theme-body-background) 23.94%,
+					var(--theme-border-white-high-contrast) 200%
+				);
+				margin-inline-start: $hero-padding-xs;
+			}
+		}
+	}
+
+	&.hero-with-details {
```

Comment:
I believe this top level component can be removed entirely, in favor of modifying the hero container on tablet and above:

```scss
.hero {
	@extend %layout-gap;
	display: flex;
	position: relative;
	flex-direction: column;
	align-items: stretch;
	justify-content: space-between;

	@include tablet {
		flex-direction: row;
	}
	//...etc